accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx

```diff
@@ -498,16 +498,16 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
                   <p style={{ marginTop: "0rem" }}>
                     The scan result displays the total number of issues found
                     with individual counts for violations, items that need
-                    review, and recommendations. By default, the issues are
+                    review, and recommendations in the issue count region. By default, the issue list is
                     shown in the ‘Element roles’ view (see the previous
                     definition), while ‘Checklist’ and ‘Rules’ tabs are also
                     available. All views show the same set of issues.
                   </p>
                 </li>
                 <li>
                   <p style={{ marginTop: "0rem" }}>
-                    To <strong>filter</strong> issues by violations, needs review or by recommendations, press any issue type toggle button in the scan results.
-                    Press the same issue type button again to see all issue types.
+                    To <strong>filter</strong> issues and focus only on violations, needs review or by recommendations, select the issue type in the issue count region below the 'Scan' button.
```

Comment:
```suggestion
                    To <strong>filter</strong> issues and focus only on violations, items that need review or recommendations, select the issue type in the issue count region below the 'Scan' button.
```
(For readability and consistency with line 500/501.)